spylib/token.py

```diff
@@ -216,14 +238,42 @@ async def execute_gql(
         return jsondata['data']
 
 
-class UniqueStore(Store):
-    __instance = None
+class OfflineTokenABC(Token, ABC):
+    """
+    Offline tokens are used for long term access, and do not have a set expiry.
+    """
+
+    @abstractmethod
+    async def save_token(self):
```

Comment:
Move that one to Token since it doesn't have arguments that depend on Online vs Offline token